src/components/znkExercise/core/directive/znkSwiperDrv.js

```diff
@@ -81,9 +81,12 @@
                         });
                     }
 
-                    function postLink(scope,element){
+                    function postLink(scope,element,attrs,ngModelCtrl){
                         $timeout(function(){
-                            defer.resolve(new Swiper(element[0]));
+                            var currSlideIndex = ngModelCtrl.$viewValue;
+                            defer.resolve(new Swiper(element[0], {
+                                initialSlide: currSlideIndex || 0
```

Comment:
starts at first slide. it's just a precaution cause always should be currSlideIndex